rust/src/mime/mime.rs

```diff
@@ -0,0 +1,605 @@
+/* Copyright (C) 2021 Open Information Security Foundation
```

Comment:
2023? Or did you write this initially in 2021?

Reply:
I wrote it in 2021 ...